scripts/generate_codesize_stats.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
+# generate_codesize_stats.sh
```

Comment:
Some name nitpicking: for now all `scripts/generate_xxx` scripts generate something that's part of the library, and this is different - should we call it `format_codesize_stats.sh` or `extract_codesize_stats.sh` instead? ("Extract" is what the usage message says.)

Reply:
Agreed - renaming to `extract_codesize_stats.sh`